CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `Genesys` source to library.
 - Fixed a bug in `Databricks.create_table_from_pandas()`. The function that converts column names to snake_case was not used in every case. (#672)
 - Added `howto_migrate_sources_tasks_and_flows.md` document. This document will assist the DEs with the viadot 1 -> viadot 2 migration process.
-
+- Fixed a bug in `Databricks.create_table_from_pandas()`. The `object` type in the DataFrame allows to store mixed values in the column which was causing the error. Added changing type of cloumn content to string. (#681)
```

Comment:
It's not clear to me what it means; whether you added a param to do that or it's done under the hood.

I would put it like that to make it clear this is done automatically: "DataFrame columns of type `object` are now automatically converted to `str`"

Reply:
[📝 Corrected entry in CHANGELOG.md](https://github.com/dyvenia/viadot/pull/682/commits/00d676c9368d78b5a7abbbf9fc6d013a690ed853)